src/main/kotlin/racingcar/car/Name.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.car
+
+data class Name(val value: String) {
+    init {
+        require(value.length <= 5) { "Car name cannot be longer than 5 characters" }
+        require(value.isNotEmpty()) { "Car name cannot be empty" }
+        require(!value.contains("\\s".toRegex())) { "Car name cannot contain whitespaces" }
+    }
```

Comment:
```suggestion
    init {
        require(value.length <= 5) { "Car name cannot be longer than 5 characters" }
        require(value.isNotEmpty()) { "Car name cannot be empty" }
        require(!value.contains(" ")) { "Car name cannot contain whitespaces" }
    }
```

What about simply this?

Reply:
Good idea!
Just so you know, what I wanted was for this condition to cover other whitespaces, like tabs.

Reply:
oh, didn't know that!